consensus/qbft-core/src/main/java/org/hyperledger/besu/consensus/qbft/core/statemachine/QbftBlockHeightManager.java

```diff
@@ -97,7 +96,11 @@ public QbftBlockHeightManager(
     this.roundFactory = qbftRoundFactory;
     this.validatorProvider = validatorProvider;
     this.transmitter =
-        new QbftMessageTransmitter(messageFactory, finalState.getValidatorMulticaster());
+        new QbftMessageTransmitter(
+            messageFactory,
+            (finalState.isFastRecoveryEnabled()
+                ? finalState.getFrequentRCMulticaster()
+                : finalState.getValidatorMulticaster()));
```

Comment:
Rather than exposing the `isFastRecoveryEnabled` and `getFrequentRCMulticaster` I think we could just construct the `QbftMessageTransmitter` in `QbftBesuControllerBuilder` configured with the correct multicaster.